src/test/java/qna/domain/AnswerTest.java

```diff
@@ -1,6 +1,101 @@
 package qna.domain;
 
+import static org.assertj.core.api.Assertions.*;
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.BDDMockito.*;
+import java.util.List;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+
+@DataJpaTest
 public class AnswerTest {
     public static final Answer A1 = new Answer(UserTest.JAVAJIGI, QuestionTest.Q1, "Answers Contents1");
     public static final Answer A2 = new Answer(UserTest.SANJIGI, QuestionTest.Q1, "Answers Contents2");
+
+    @Autowired
+    AnswerRepository answerRepository;
+
+    @MockBean
+    UserRepository userRepository;
+
+    @MockBean
+    QuestionRepository questionRepository;
+
+    @BeforeEach
+    void setUp() {
+        목업_자바지기와_상지기_유저등록(UserTest.JAVAJIGI);
```

Comment:
```suggestion
        유저등록(UserTest.JAVAJIGI);
```

으로 충분히 의도를 표현할 수 있지 않을까요!?

Reply:
말씀대로 바꾸면 좋을거같습니다. 😄 
처음에는 파라미터없이 해당 두 유저를 생성하도록 만들었던 메서드라 스펙 변경 후에 메서드명 또한 변경해줬어야 했는데, 놓친거같습니다 😱